joomla/joomla-cms
# [Regression] Clean cache of installed language query on language extension install/update/remove

Pull Request for New Issue.

### Summary of Changes

After https://github.com/joomla/joomla-cms/pull/12671 when you have cache enabled an install a language the language cannot be used until you clean the cache.

This PR intends to solve that.

### Testing Instructions

1. Use latest staging
2. Enable cache
3. Install a language and go to installed languages see the installed language is not there.
4. Clean the cache. the language is there now.
5. Apply patch
6. Install a language. Repeat step 3. the language is there.

### Documentation Changes Required

None.
